components/openzwave-adapter/src/lib.rs

```diff
@@ -439,7 +491,7 @@ impl taxonomy::adapter::Adapter for OpenzwaveAdapter {
 
                 ozw_vid_as_taxo_value(&ozw_vid)
             });
-            let value_result: Result<Option<Value>, TaxoError> = taxo_value.ok_or(TaxoError::InternalError(InternalError::NoSuchGetter(id.clone())));
+            let value_result: Result<Option<Value>, TaxoError> = taxo_value.ok_or(TaxoError::GetterDoesNotSupportPolling(id.clone()));
```

Comment:
this will likely be an id that is not a value_id... Likely we'll have to do better later on.